openml-datarobot/README.md

```diff
@@ -55,11 +55,10 @@ To import DataRobot models you need to place them in a special folder structure
 }
 ```
 
-This is an example of a schema definition for a model.
-If you don't provide a "model.json" file for a model you will need to create the schema and add it through the model import UI.
+If you don't provide a "model.json" file for a model, you will need to create the schema and add it during the model import steps in the UI.
 
 Notes:
-- The field indices ("fieldIndex") for fieldSchemas are required and start at 0.
+- The field indexes ("fieldIndex") for fieldSchemas are required and start at 0.
 - Field schemas must be ordered by ascending field index.
 - The target index ("targetIndex") must be a valid index pointing to the field schema of the target variable of the model.
 - The available value schema types for DataRobot models are, as shown, "categorical", "string" and "numeric".
```

Comment:
Please explicit that we need DataRobot models exported by a "recent enough" version of the DataRobot platform. We don't know how to quantify the exact version. This is a problem if the user tries to load a model and gets the following error:

`Jar file of the DataRobot model is not supported. A possible cause is that the model might be to old and a newer version is required because it lacks the "classLabes" field with the target values. If that is the cause create a new project on DataRobot and train new models.`